.github/workflows/integration-tests-benchmarks.yml

```diff
@@ -77,6 +79,9 @@ jobs:
           java-version: '17'
           distribution: 'temurin'
 
+      - name: Setup Gradle
```

Comment:
Do we need this again?

Reply:
yep, it's a different job that runs in parellel with the one above, so it also has to go through the full setup afaik

Reply:
ah yeah, i didn't read the lines in between 😅